src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -3576,6 +3576,7 @@
         Assert.Null(comp.GetTypeByMetadataName("<>F1__C"));
         Assert.Null(comp.GetTypeByMetadataName("F0__C"));
         Assert.Null(comp.GetTypeByMetadataName("<file>F0__C"));
+        Assert.Null(comp.GetTypeByMetadataName("C"));
```

Comment:
>Assert.Null(comp.GetTypeByMetadataName("C")); [](http://example.com/codeflow?start=8&length=45)

Please also assert:
```
Assert.Null(comp.GetTypeByMetadataName("C`1"));
```

Reply:
> I am not following the logic of why you requested such assert exactly in this place, since there is only a non-generic version of type C in the scope. 

To make sure that generic mangling on the name produces expected result for this scenario.